mitodl/mitxpro
# Upgrade uwsgi

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
This upgrades uwsgi to potentially address some deadlocking issues we're seeing in deployed environments.

#### How should this be manually tested?
- `docker-compose build`
- Ensure the app runs and is functional (you can hit the site, cms pages, login is probably adequate)
